src/libs/actions/Welcome.js

```diff
@@ -118,9 +119,13 @@ function show({routes, showCreateMenu}) {
         const isDisplayingWorkspaceRoute = isWorkspaceRoute || exitingToWorkspaceRoute;
 
         // We want to display the Workspace chat first since that means a user is already in a Workspace and doesn't need to create another one
-        const workspaceChatReport = _.find(allReports, report => ReportUtils.isPolicyExpenseChat(report) && report.ownerEmail === email);
+        const workspaceChatReport = _.find(
+            allReports,
+            report => ReportUtils.isPolicyExpenseChat(report) && report.ownerEmail === email && report.statusNum !== CONST.REPORT.STATUS.CLOSED,
```

Comment:
Why do we need `report.statusNum !== CONST.REPORT.STATUS.CLOSED`? Were we redirecting users to closed policy chat reports?

Reply:
Ah just saw this comment in the issue - https://github.com/Expensify/Expensify/issues/270487#issuecomment-1498038852

Reply:
All good here